frontend/public/locales/en/translation.json

```diff
@@ -409,6 +410,18 @@
     "title": "My comparisons",
     "listHasNbComparisons_many": "You already did <1>{{comparisonCount}}</1> comparisons."
   },
+  "criteriaDescriptions": {
+    "backfire_risk": "<0>Is it adapted to viewers with opposing beliefs? Does it prevent misconceptions or undesirable reactions?</0><1>This criterion aims to measure the probability that a user may misunderstand or negatively react to the information presented. This may be particularly likely to occur for contents presented with a strong \"us versus them\" atmosphere on controversial topics<1>(<1>BailABBC+-18</1>)</1>. Note, however, that this <4>Veritasium video</4> (based on his <8>PhD thesis</8>) shows that backfiring also occurs for clean presentation of a counter-intuitive scientific concept, if the content does not pinpoint the risks of misunderstandings.</1><2>Contents that score high on \"resilience to backfiring risks\" should be safe to recommend to all sorts of viewers, with limited risks of misunderstandings.</2>",
+    "better_habits": "<0>Does it make people adopt habits that benefit themselves and beyond?</0><1>This criterion aims to highlight content that nudges viewers towards better habits, for themselves or for society. These habits may include healthier habits, in terms of food, sport activities or mental health, or more careful thinking when consuming informational contents. They may also include more altruistic habits, such as being kinder to others (including on the Internet), reducing our carbon footprint and being wary of our overconfidence. Evidently, contents that do not promote better habits should be given lower ratings, while contents that promote poor habits, purposely or not, should be given very low ratings.</1><2>Contents that score high on \"encourages better habits\" should be successful at motivating viewers to grow and improve themselves.</2>",
```

Comment:
> [...] This criterion aims to highlight content that nudges viewers towards better habits [...]

Should the content be plural here? Or maybe we are talking about which video of the two being compared is encouraging better habits?